madgraph/iolibs/template_files/pdf_wrap_emela.f

```diff
@@ -0,0 +1,103 @@
+      subroutine pdfwrap
+      implicit none
+C
+C     INCLUDE
+C
+      include 'pdf.inc'
+      include '../alfas.inc'
+      include '../coupl.inc'
+      real*8 zmass
+      data zmass/91.188d0/
+      Character*150 LHAPath
+      character*20 parm(20)
+      double precision value(20)
+      real*8 alphasPDF
+      external alphasPDF
+      ! PDFs with beamstrahlung use specific initialisation/evaluation
+      logical has_bstrahl
+      common /to_has_bs/ has_bstrahl
+
+
+c-------------------
+c     START THE CODE
+c-------------------      
+
+c     initialize the pdf set
+      call FindPDFPath(LHAPath)
+      CALL SetPDFPath(LHAPath)
+      value(1)=lhaid
+      parm(1)='DEFAULT'
+      if (pdlabel.eq.'emela') then
+         if (has_bstrahl) then
+            call bs_initfromgrid_lhaid(LHAID)
+         else
+            call initfromgrid_lhaid(lhaid)
+         endif
+         nloop = 2 
+         asmz = g**2/16d0/datan(1d0) 
+      else
+          write(*,*) 'Unknown PDLABEL', pdlabel
+          stop 1
+      endif
+      
+      return
+      end
+ 
+
+      subroutine FindPDFPath(LHAPath)
```

Comment:
Don't understand the comment

Reply:
It is just that this sounds a function similar to an existing one.
After searching, I found this one:
Source/open_file.f
Which is certainly not the same function since that one open the file in top of it.
But this might be a good idea to not repeat ourself.